src/components/ManageTask/ManageTask.tsx

```diff
@@ -1,7 +1,8 @@
 import Modal from "react-bootstrap/Modal";
 import Button from "react-bootstrap/esm/Button";
 import React, {Component, FormEvent} from "react";
-import avatar from '../Task/task.svg';
+import { Image} from '@material-ui/icons';
```

Comment:
We use only FontAwesome. Please investigate if this module is used on this file.